packages/edit-site/src/components/page-patterns/no-patterns.js

```diff
@@ -2,11 +2,48 @@
  * WordPress dependencies
  */
 import { __ } from '@wordpress/i18n';
+import {
+	Button,
+	Icon,
+	__experimentalVStack as VStack,
+	__experimentalHeading as Heading,
+	__experimentalText as Text,
+} from '@wordpress/components';
+import { symbol } from '@wordpress/icons';
+
+/**
+ * Internal dependencies
+ */
+import AddNewPattern from '../add-new-pattern';
 
 export default function NoPatterns() {
 	return (
-		<div className="edit-site-patterns__no-results">
-			{ __( 'No patterns found.' ) }
-		</div>
+		<VStack
+			spacing={ 6 }
+			alignment="center"
+			className="edit-site-patterns__no-results"
+		>
+			<Icon icon={ symbol } />
+			<VStack spacing={ 3 } alignment="center">
+				<Heading level={ 4 } as="h2">
+					{ __( 'Create a pattern' ) }
```

Comment:
I wonder if we should still show text suggesting no patterns for accessibility reasons?